doc/release/README.bugs.rst

```diff
@@ -18,15 +18,16 @@ in confidence by the Chapel team, please mail it to:
 Please include as much information as possible in your email.  For
```

Comment:
I agree it would be nice to split it up. If I did, I'd have to merge in the README conversion before this.

I needed at least 1 README from `doc/release/`, otherwise sphinx would throw a fit because `doc/release/sphinx/source/index.rst` wouldn't find any .rst files.